test/e2e/storage/flexvolume.go

```diff
@@ -152,7 +152,7 @@ func getHostFromHostPort(hostPort string) string {
 	return host
 }
 
-var _ = utils.SIGDescribe("Flexvolumes", func() {
+var _ = utils.SIGDescribe("[Feature:Flexvolumes]", func() {
```

Comment:
The "should be mountable when non-attachable" test case below does not require installing the driver on the master node, so I think we should still run that one by default to test node flex functionality.

Reply:
@msau42 added the `Feature:Flexvolumes` to just "mountable when attachable" test so the other one should run now